docs/mosviz/import_data.rst

```diff
@@ -50,8 +67,6 @@ and for NIRISS:
 
     jdaviz mosviz /path/to/my/data --instrument=niriss
 
-If an instrument is not specified in either case, Mosviz will default to NIRSpec parsing.
```

Comment:
I think we want to keep this sentence but tell the user now that it would throw an error.